ow_power_system/include/PowerSystemPack.h

```diff
@@ -29,6 +35,33 @@ using PrognoserVector = std::vector<PrognoserMap>;
 //       so it may not be addressable here.
 const int NUM_NODES = 8;
 
+// Change this value to modify the number of samples each node creates during the
+// Monte Carlo prediction process. Lower values mean faster performance, but lower
+// accuracy (needs testing for confirmation).
+// The default value is 100, but this is very slow and infeasible for the simulation.
+// My computer reached similar speeds to the original simple prognoser at a sample
+// value of 25, but this may vary from computer to computer, and it also varies
+// depending on the value of the RUL prediction. Higher RULs take significantly longer
+// to return. ~Liam
+const int NUM_SAMPLES = 100;
+
+// Change this value to modify the maximum RUL estimation output from the Monte
+// Carlo prediction process. Lower values mean faster performance in the event
+// a RUL prediction would exceed the horizon value, but it also means the prognoser
+// can't return RUL values higher than this.
+// The default value is 10000, but this is slow in situations where the prognoser
+// predicts a value that high (or higher than that). Testing and discussion
+// is needed to determine the ideal value for this.
+const int MAX_HORIZON = 10000;
```

Comment:
Should specify this is seconds, or better yet include it in the name, e.g. MAX_HORIZON_SECS.

Outside the scope of this PR, but I don't see how we can do useful prognosis with a value even this small (less than 3 hours) given the timeline of the mission.

Reply:
The horizon isn't the only value that affects GSAP performance, at least; sample count is significantly more important. The problem is that we don't really know how big of a dip in accuracy results from much lower sample counts, but if we need RUL predictions in the realm of ~20 hours (72000 seconds) I wouldn't be surprised if we have to reduce sample count to 10 or even lower.

The other option could be increasing the profile/timestamp increment to something longer, like 5 or even 10, so instead of requiring GSAP to return 24 predictions in 2 seconds it could be every 5 or 10 seconds. This would require a lot of updating across the power system and possibly other components, though.